pkg/component/shared/plutono.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shared
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/gardener/gardener/pkg/component"
+	"github.com/gardener/gardener/pkg/component/plutono"
+	"github.com/gardener/gardener/pkg/utils/images"
+	"github.com/gardener/gardener/pkg/utils/imagevector"
+	secretsmanager "github.com/gardener/gardener/pkg/utils/secrets/manager"
+)
+
+// NewPlutono returns a deployer for the plutono.
+func NewPlutono(
+	c client.Client,
+	namespace string,
+	imageVector imagevector.ImageVector,
+	secretsManager secretsmanager.Interface,
+	authSecretName string,
+	clusterType component.ClusterType,
+	ingressHost string,
+	includeIstioDashboards, gardenletManagesMCMbool, nodeLocalDNSEnabled, isWorkerless, vpnHighAvailabilityEnabled bool,
```

Comment:
```suggestion
	includeIstioDashboards, gardenletManagesMCM, nodeLocalDNSEnabled, isWorkerless, vpnHighAvailabilityEnabled bool,
```

Minor: Is the bool suffix needed?

Reply:
Thank you, @istvanballok for reviewing and testing all these scenarios. I  have addressed the comment in 0595561b8377596980f6905100927a1f3ba17836. PTAL